messages/project.deploy.pipeline.quick.md

```diff
@@ -4,7 +4,9 @@ Quickly deploy a validated deployment to an org.
 
 # description
 
-Before you run this command, first create a validated deployment with the "sf project deploy pipeline validate" command, which returns a job ID. Validated deployments haven't been deployed to the org yet; you deploy them with this command. Either pass the job ID to this command or use the --use-most-recent flag to use the job ID of the most recently validated deployment. For the quick deploy to succeed, the associated validated deployment must also have succeeded.
+The first time you run any "project deploy pipeline" command, be sure to authorize the org in which DevOps Center is installed. The easiest way to authorize an org is with the "sf org login web" command.
```

Comment:
```suggestion
The first time you run any "project deploy pipeline" command, be sure to authorize the org in which DevOps Center is installed. The easiest way to authorize an org is with the "org login web" command.
```

Reply:
I tend to not include the "sf" part when I mention commands, because it's really the binary, not the command.

Reply:
Although I'm sure a few "sf"s have slipped in now and again.  I think it's really okay, no biggie.